crates/cargo-contract/src/cmd/extrinsics/call.rs

```diff
@@ -108,7 +109,7 @@ impl CallCommand {
 
         let signer = super::pair_signer(self.extrinsic_opts.signer()?);
 
-        async_std::task::block_on(async {
+        Runtime::new().unwrap().block_on(async {
```

Comment:
Please replace this and other `Runtime::new().unwrap()` occurrences with `?`, using `map_err` where necessary.